armi/nucDirectory/nuclideBases.py

```diff
@@ -91,44 +91,45 @@ class which is used to organize and store metadata about each nuclide. The
 >>> nuclideBases.byAAAZZZSId['2350920']
 <NuclideBase U235:  Z:92, A:235, S:0, W:2.350439e+02, Label:U235>, HL:2.22160758861e+16, Abund:7.204000e-03>
 
-.. _nuclide-bases-table:
-
-.. exec::
-    import numpy
-    from tabulate import tabulate
-    from armi.nucDirectory import nuclideBases
-    
-    attributes = ['name',
-                  'type',
-                  'a',
-                  'z',
-                  'state',
-                  'abundance',
-                  'weight',
-                  'halflife']
-    
-    def getAttributes(nuc):
-        if nuc.halflife == numpy.inf:
-            halflife = "inf"
-        else:
-            halflife = f'{nuc.halflife:<12.6e}'
-        return [
-            f'``{nuc.name}``',
-            f':py:class:`~armi.nucDirectory.nuclideBases.{nuc.__class__.__name__}`',
-            f'``{nuc.a}``',
-            f'``{nuc.z}``',
-            f'``{nuc.state}``',
-            f'``{nuc.abundance:<12.6e}``',
-            f'``{nuc.weight:<12.6e}``',
-            f'``{halflife}``',
-        ]
-    
-    sortedNucs = sorted(nuclideBases.instances)
-    return create_table(tabulate(tabular_data=[getAttributes(nuc) for nuc in sortedNucs],
-                                    headers=attributes,
-                                    tablefmt='rst'),
-                                    caption='List of nuclides')
+.. only:: html
```

Comment:
Did we test this works in the PDF?

Reply:
Yes. It doesn't show up in the PDF, but it is still there in the HTML